packages/logger/logger.pony

```diff
@@ -97,15 +98,19 @@ type LogLevel is
 
 primitive Fine
   fun apply(): U32 => 0
+  fun string(): String iso^ => recover String(4).>append("FINE") end
```

Comment:
this doesn't need String(4) etc, the constant 

"FINE"

will work.

Reply:
I'm getting compilation errors on 0.35.1:
```
/home/richard/code/pony/ponyc/packages/logger/logger.pony:101:32: can't recover to this capability
  fun string(): String iso^ => recover iso "FINE" end
                               ^
    Info:
    /home/richard/code/pony/ponyc/packages/logger/logger.pony:101:44: expression type is String val
      fun string(): String iso^ => recover iso "FINE" end
                                               ^
```

I went with trying to return `String iso^` so LogLevel implements the Stringable interface. I don't see much harm in returning `String val`, though.

Reply:
This is how we currently do in the rest of the standard library:

https://playground.ponylang.io/?gist=4d72cefd46cf9a24ba5e78d4e6e597de

So let's go with that for consistency.

Here's some examples:

primitive ExecveError
  fun string(): String iso^ => "ExecveError".clone()

primitive PipeError
  fun string(): String iso^ => "PipeError".clone()

primitive ForkError
  fun string(): String iso^ => "ForkError".clone()

Reply:
Ah, yes. That is much nicer.

Done.